types/marked/index.d.ts

```diff
@@ -122,86 +113,82 @@ declare namespace marked {
+            inRawBlock: boolean,
+            smartypants: (cap: string) => string,
+        ): NonNullable<Tokens.Text | T>;
     }
 
-    class Renderer {
+    type TokenizerObject = Partial<Omit<Tokenizer<false>, "constructor" | "options">>;
+
+    class Renderer<T = null> {
         constructor(options?: MarkedOptions);
         options: MarkedOptions;
-        code(code: string, language: string | undefined, isEscaped: boolean): string;
-        blockquote(quote: string): string;
-        html(html: string): string;
-        heading(text: string, level: 1 | 2 | 3 | 4 | 5 | 6, raw: string, slugger: Slugger): string;
-        hr(): string;
-        list(body: string, ordered: boolean, start: number): string;
-        listitem(text: string): string;
-        checkbox(checked: boolean): string;
-        paragraph(text: string): string;
-        table(header: string, body: string): string;
-        tablerow(content: string): string;
+        code(code: string, language: string | undefined, isEscaped: boolean): NonNullable<string | T>;
+        blockquote(quote: string): NonNullable<string | T>;
+        html(html: string): NonNullable<string | T>;
+        heading(text: string, level: 1 | 2 | 3 | 4 | 5 | 6, raw: string, slugger: Slugger): NonNullable<string | T>;
+        hr(): NonNullable<string | T>;
+        list(body: string, ordered: boolean, start: number): NonNullable<string | T>;
+        listitem(text: string): NonNullable<string | T>;
+        checkbox(checked: boolean): NonNullable<string | T>;
+        paragraph(text: string): NonNullable<string | T>;
+        table(header: string, body: string): NonNullable<string | T>;
+        tablerow(content: string): NonNullable<string | T>;
         tablecell(
             content: string,
             flags: {
                 header: boolean;
-                align: 'center' | 'left' | 'right' | null;
+                align: "center" | "left" | "right" | null;
             },
-        ): string;
-        strong(text: string): string;
-        em(text: string): string;
-        codespan(code: string): string;
-        br(): string;
-        del(text: string): string;
-        link(href: string | null, title: string | null, text: string): string;
-        image(href: string | null, title: string | null, text: string): string;
-        text(text: string): string;
+        ): NonNullable<string | T>;
+        strong(text: string): NonNullable<string | T>;
+        em(text: string): NonNullable<string | T>;
+        codespan(code: string): NonNullable<string | T>;
+        br(): NonNullable<string | T>;
+        del(text: string): NonNullable<string | T>;
+        link(href: string | null, title: string | null, text: string): NonNullable<string | T>;
+        image(href: string | null, title: string | null, text: string): NonNullable<string | T>;
+        text(text: string): NonNullable<string | T>;
     }
 
+    type RendererObject = Partial<Omit<Renderer<boolean>, "constructor" | "options">>;
```

Comment:
Shouldn't it be `false` instead of `boolean`?
```suggestion
    type RendererObject = Partial<Omit<Renderer<false>, "constructor" | "options">>;
```